docs/src/content/docs/it/reference/overrides.md

```diff
@@ -332,6 +345,12 @@ Componente contenente l'elemento `<h1>` per la pagina corrente.
 
 Le implementazioni dovrebbero garantire di impostare `id="_top"` sull'elemento `<h1>` come nell'implementazione predefinita.
 
+#### `DraftContentNotice`
+
+**Componente predefinito:** [`DraftContentNotice.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/DraftContentNotice.astro)
```

Comment:
It looks all other instances in this page are using "Componente standard" instead of "Componente predefinito". Not sure if one is better than the other, but I think they should be consistent. Do you think they should all be updated?

Reply:
No, I think "Componente standard" should be used, will update it